### DISCUSSION TITLE
[Feature] Ability to zoom out to year view

### DISCUSSION THREAD

**Discussion by frejaya** [immich-app/immich#8043] [Feature Request]
### The feature

Right now we can use pinch or assets per row setting plus group by month setting to get a pretty high-level view of photos. I would like to be able to go further and zoom out to a year view. I would be fine if when viewing in year view, some photos aren't shown for performance. I just want that really extreme at a glance of decades of photos because I have photos starting over a hundred years ago and seeing them progress over time is nice. I am able to do this in Synology Photos and it's really nice. Not a huge priority, month view has been good enough, just a nice to have!

### Platform

- [ ] Server
- [X] Web
- [X] Mobile

**Comment by DaveTSG** [comment]
Is this the sort of thing you mean? #4455
